documentation/healthcare-analytics-workspace.adoc

```diff
@@ -0,0 +1,471 @@
+== Healthcare Analytics
+:images: {img}
+:data: https://neo4j-graph-examples.github.io/healthcare-analytics/data
+
+image::{img}/healthcare-analytics.jpg[float=right,width=200px]
+
+Health care analytics allows for the examination of patterns in healthcare data to determine how clinical care can be improved. 
+
+Healthcare organizations can _"realize new opportunities and efficiencies by leveraging the connections within their existing data: be it in a connected genome, or a provider network, or patient treatments,"_.
```

Comment:
Sounds almost too much like marketing. I'd paraphrase it and leave Emil's name out of it.

Reply:
I think you are right. I left this in because I got down the original guide so much...